### ISSUE TITLE
Blank white screen in content manager pages

### ISSUE THREAD

**Issue by bumbeishvili** [strapi/strapi#15077]
## Content Manager page is breaking in the deployed version

We are facing a strange issue that we are not able to fix for days now.

In the deployed version of Strapi, when trying to visit the content manager page, it breaks and the white page appears. The error message is not helpful to understand what's happening.

Everything works nicely locally, but in the deployed version, the content manager part is breaking. 

https://user-images.githubusercontent.com/6873202/205608887-3e4cdf64-89b8-4ed3-a2b6-16a62b8ab5d1.mp4

**What we tried so far**:
* Removing all records and then trying deployments this way
* Switching between different versions of Strapi
* Deploying it into a completely new environment with an empty database

### Required System information

- Node.js version:`>=14.19.1 <=18.x.x`
- NPM version:`>=6.0.0`
- Strapi version: `4.5.3`
- Database:`Postgress`
- Operating system: `Local MacOs` ,  `Deployed version Linux`

### Describe the bug

In the deployed version of Strapi CMS, when visiting Content Manager, page breaks and blank white page appears

### Steps to reproduce the behavior

1. Click on `Content Manager`
2. See error
```
content-manager.2aaa6d0e.chunk.js:70 TypeError: Cannot read properties of undefined (reading 'reduce')
    at pu (content-manager.2aaa6d0e.chunk.js:70:7442)
    at mu (content-manager.2aaa6d0e.chunk.js:70:7814)
    at content-manager.2aaa6d0e.chunk.js:70:12428
    at Generator.next (<anonymous>)
    at l (content-manager.2aaa6d0e.chunk.js:70:11818)
(anonymous) @ content-manager.2aaa6d0e.chunk.js:70
l @ content-manager.2aaa6d0e.chunk.js:70
Promise.then (async)
u @ content-manager.2aaa6d0e.chunk.js:70
(anonymous) @ content-manager.2aaa6d0e.chunk.js:70
wu @ content-manager.2aaa6d0e.chunk.js:70
(anonymous) @ content-manager.2aaa6d0e.chunk.js:70
(anonymous) @ content-manager.2aaa6d0e.chunk.js:70
zc @ main.ab455542.js:10921
p.unstable_runWithPriority @ main.ab455542.js:10956
cu @ main.ab455542.js:10917
Xi @ main.ab455542.js:10921
Qs @ main.ab455542.js:10919
(anonymous) @ main.ab455542.js:10917
p.unstable_runWithPriority @ main.ab455542.js:10956
cu @ main.ab455542.js:10917
fl @ main.ab455542.js:10917
ti @ main.ab455542.js:10917
$l @ main.ab455542.js:10919
notify @ main.ab455542.js:10932
u @ main.ab455542.js:10932
c @ main.ab455542.js:10932
Q @ main.ab455542.js:10949
(anonymous) @ main.ab455542.js:10565
(anonymous) @ main.ab455542.js:10565
(anonymous) @ main.ab455542.js:10565
(anonymous) @ main.ab455542.js:10565
(anonymous) @ content-manager.2aaa6d0e.chunk.js:1178
l @ content-manager.2aaa6d0e.chunk.js:1178
Promise.then (async)
u @ content-manager.2aaa6d0e.chunk.js:1178
l @ content-manager.2aaa6d0e.chunk.js:1178
Promise.then (async)
u @ content-manager.2aaa6d0e.chunk.js:1178
(anonymous) @ content-manager.2aaa6d0e.chunk.js:1178
$C @ content-manager.2aaa6d0e.chunk.js:1178
m @ content-manager.2aaa6d0e.chunk.js:1178
(anonymous) @ content-manager.2aaa6d0e.chunk.js:1178
zc @ main.ab455542.js:10921
p.unstable_runWithPriority @ main.ab455542.js:10956
cu @ main.ab455542.js:10917
Xi @ main.ab455542.js:10921
(anonymous) @ main.ab455542.js:10921
U @ main.ab455542.js:10956
C.port1.onmessage @ main.ab455542.js:10956
main.ab455542.js:10919 TypeError: Cannot read properties of undefined (reading 'editRelations')
    at Rr (content-manager.2aaa6d0e.chunk.js:988:13380)
    at ns (main.ab455542.js:10917:54832)
    at Uc (main.ab455542.js:10921:9232)
    at Lc (main.ab455542.js:10921:992)
    at nd (main.ab455542.js:10921:920)
    at vs (main.ab455542.js:10921:771)
    at Qs (main.ab455542.js:10919:11158)
    at main.ab455542.js:10917:39629
    at p.unstable_runWithPriority (main.ab455542.js:10956:3787)
    at cu (main.ab455542.js:10917:39407)
```

### Expected behavior

The content Manager page should not break

### Screenshots

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/6873202/205611019-9907ae92-a49e-405a-acba-a14637d7945c.png">

### Additional context

We exhausted all options and still have no idea what may be causing it. 

Does anyone have or faced a similar issue in the past? What was causing factor and how you were able to solve it?  Is there a way to show a better error message, which might push us into correct direction of solving this issue?

**Comment by gu-stav** [comment]
Is it possible you've upgraded from version < v4.5 to version > v4.5? `editRelations` shouldn't exist anywhere in the codebase anymore. You can try to delete the `.cache` and `build` folder and rebuild the admin interface using `yarn build`. This should fix the issue.

**Comment by iwarner** [comment]
@gu-stav @bumbeishvili 

Version: 4.5.3
DN: SQL Lite - but same on Postgres(Prod)
Node: 16

I am facing the same issues - with certain blocks in a dynamic zones, that have relations with other collections.

Please check the video on this matter
https://sharing.clickup.com/clip/p/t2415269/366aead7-af90-4b8e-969a-64a456c138a9/screen-recording-2022-12-05-17%3A19.webm

Console error

`TypeError: i is not iterable
    at content-manager.1c8905d1.chunk.js:1013:19226
    at G.produce (main.dbaab4af.js:24:43809)
    at vh (content-manager.1c8905d1.chunk.js:1013:17970)
    at Object.Qo [as useReducer] (main.dbaab4af.js:1438:55851)
    at t.useReducer (main.dbaab4af.js:1463:5864)
    at Nr (content-manager.1c8905d1.chunk.js:1013:23167)
    at tl (main.dbaab4af.js:1438:54584)
    at ol (main.dbaab4af.js:1438:64350)
    at lr (main.dbaab4af.js:1442:10106)
    at Xc (main.dbaab4af.js:1442:990)`

**Comment by bumbeishvili** [comment]
> Is it possible you've upgraded from version < v4.5 to version > v4.5? editRelations shouldn't exist anywhere in the codebase anymore.

I think yes, we did upgrade and then downgrade the version after facing new issues

>  You can try to delete the .cache and build folder and rebuild the admin interface using yarn build. This should fix the issue.

Already doing this using docker, no change

**Comment by iwarner** [comment]
@bumbeishvili https://sharing.clickup.com/clip/p/t2415269/4f26cf95-5765-40c5-bb2d-8e57d816c5dc/screen-recording-2022-12-05-17%3A39.webm

Another video with clean DB

**Comment by gu-stav** [comment]
Could you try and build Strapi in development mode (so that we can see the source maps) or share the affected code in `content-manager.[id].chunk.js` with us? With modification it is almost impossible to understand, what went wrong.

Also ping @joshuaellis here.

**Comment by alimpens** [comment]
@iwarner, I think your issue is the same as [this issue](https://github.com/strapi/strapi/issues/14908), we ran into it as well in our project.

**Comment by joshuaellis** [comment]
This should be solved with `4.6.0` if your issue persists please open a new issue and follow the template.

**Comment by bumbeishvili** [comment]
It was solved, thank you

**Comment by ahyanamoworks** [comment]
@bumbeishvili do you remember how did you resolved it?

**Comment by bumbeishvili** [comment]
By updating Strapi

**Comment by abhijithkmpixel** [comment]
I still i have a black white screen when using internationalization!
when switching to another locale inside a collection type the page goes completely white
not only once but most times it is present

Node: v18.17.0
npm: 9.6.7
windows 11 home 64 bit
"@strapi/strapi": "4.25.4",

please let me know when or how this can be fixed
